plugins/data_migration/api/data_migration_helper.js

```diff
@@ -550,35 +557,35 @@ module.exports = function(my_db) {
 
                         for (let j = 0; j < drill_events.length; j++) {
                             eventCollName = "drill_events" + crypto.createHash('sha1').update(drill_events[j] + appid).digest('hex');
-                            scripts.push('mongodump ' + dbstr_drill + ' --collection ' + eventCollName + ' --out ' + my_folder);
+                            scripts.push({cmd: 'mongodump', args: [...dbargs_drill, '--collection', eventCollName, '--out', my_folder]});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['eventCollName' used out of scope.](https://app.codacy.com/app/ar2rsawseen/countly-server/pullRequest?prid=4577914)